CHANGELOG.asciidoc

```diff
@@ -28,18 +28,17 @@ https://github.com/elastic/beats/compare/v7.6.2...v7.7.0[View commits]
 *Affecting all Beats*
 
 - Fix Kubernetes autodiscovery provider to correctly handle pod states and avoid missing event data. {pull}17223[17223]
-- Fix `add_cloud_metadata` to better support modifying sub-fields with other processors. {pull}13808[13808]
+- Fix `add_cloud_metadata` processor to better support modifying sub-fields with other processors. {pull}13808[13808]
 - Fix panic in the Logstash output when trying to send events to closed connection. {pull}15568[15568]
 - Fix logging target settings being ignored when Beats are started via systemd or docker. {issue}12024[12024] {pull}15422[15442]
 - Fix issue where default go logger is not discarded when either * or stdout is selected. {issue}10251[10251] {pull}15708[15708]
 - Remove superfluous use of `number_of_routing_shards` setting from the default template. {pull}16038[16038]
 - Automatically convert index names to lowercase. {pull}16081[16081]
-- Upgrade go-ucfg to latest v0.8.1. {pull}15937{15937}
 - Fix loading processor annotation hints, allowing the value to be a full configuration section. {pull}16348[16348]
-- Upgrade go-ucfg to latest v0.8.3. {pull}16450{16450}
+- Upgrade go-ucfg to latest v0.8.3. {pull}16450[16450]
```

Comment:
I'd say users don't care about updated patch releases. This PR might be the only user visible one: https://github.com/elastic/go-ucfg/pull/139
But this might be fixed in 7.7 already.

Reply:
I agree.  I think we should remove the `go-ucfg` updates.